backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/PokerController.java

```diff
@@ -0,0 +1,106 @@
+import com.cgi.retrospecto.backend.poker.controller.dto.*;
+import com.cgi.retrospecto.backend.poker.domain.Room;
+import com.cgi.retrospecto.backend.poker.domain.Story;
+import com.cgi.retrospecto.backend.poker.domain.Vote;
+import com.cgi.retrospecto.backend.poker.exception.RoomNotFoundException;
+import com.cgi.retrospecto.backend.poker.exception.StoryNotFoundException;
+import com.cgi.retrospecto.backend.poker.helper.converter.RoomConverter;
+import com.cgi.retrospecto.backend.poker.helper.converter.StoryConverter;
+import com.cgi.retrospecto.backend.poker.service.RoomService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.web.bind.annotation.*;
+
+import static com.cgi.retrospecto.backend.poker.api.PokerConstants.ApiConstants.*;
+
+@RestController
+@RequestMapping(POKER)
+@CrossOrigin
+public class PokerController {
+    @Autowired
+    private RoomService roomService;
+
+    @ResponseBody
+    @RequestMapping(path = ROOM, method = RequestMethod.POST, consumes = MediaType.TEXT_PLAIN_VALUE)
+    public ResponseEntity<RoomDetails> createRoom(@RequestBody String author) {
+        return new ResponseEntity<>(
+                RoomConverter.toDTO(
+                        roomService.createRoom(
+                                RoomConverter.toEntity(author)
+                        )
+                ), HttpStatus.OK);
+    }
+
+    @ResponseBody
+    @RequestMapping(path = GET_ROOM, method = RequestMethod.GET)
+    public ResponseEntity<RoomDetails> getRoom(
+            @PathVariable(PokerConstants.ID) int id,
+            @PathVariable(PokerConstants.USERNAME) String username)
+            throws RoomNotFoundException {
+        Room room = roomService.getPokerRoom(id);
+        room.addUser(username);
+        return new ResponseEntity<>(RoomConverter.toDTO(room), HttpStatus.OK);
+    }
+
+    @MessageMapping(POKER + "/{roomId}/story/add")
+    @SendTo("/topic" + POKER + "/{roomId}/story/add")
+    public ResponseEntity<Story> createStory(
+            @DestinationVariable int roomId,
+            @Payload String story) throws RoomNotFoundException {
+        System.out.println("Creating story: " + story + " --> " + roomId);
+        return new ResponseEntity<>(
+                roomService.createStory(roomId,
+                        StoryConverter.toEntity(story)
+                ), HttpStatus.OK);
+    }
+
+    @MessageMapping(POKER + "/{roomId}/story/{storyId}/vote")
+    @SendTo("/topic" + POKER + "/{roomId}/vote")
+    public ResponseEntity<RoomDetails> vote(
```

Comment:
This could be improved by not returning the whole room rather just the Story or the Vote itself, then less updating us needed on the frontend and it could be a one liner.